atomic_reactor/plugins/input_osv3.py

```diff
@@ -62,6 +62,16 @@ def run(self):
 
         self.log.debug("build json: %s", input_json)
 
+        # Compatibility code for dockerfile_content plugin
+        # If this (removed) plugin is requested, ignore it.
+        if 'prebuild_plugins' in self.plugins_json:
+            for index, plugin in enumerate(self.plugins_json['prebuild_plugins']):
+                if plugin['name'] == 'dockerfile_content':
```

Comment:
`dockerfile_content` is the only one that prevents using atomic-reactor-1.6.20 with osbs-client-0.33.1. I was thinking we'd remove this work-around before 1.6.21.